README.md

```diff
@@ -310,6 +311,27 @@ In darwin, `libclang.dylib`, In Linux, `libclang.so`.
 C language standard version option.  
 If not set, deoplete-go use `c11`(latest) version.
 
+### `g:deoplete#sources#go#goos`
+#### Set GOOS environment variable when calling `gocode`
+
+| **Default**  | `''`      |
+|--------------|-----------|
+| **Required** | No        |
+| **Type**     | string    |
+| **Example**  | `windows` |
+
+Sets the `GOOS` environment variable for `gocode`.  Set it to `auto` if you
+want `GOOS` to be automatically set depending on the current buffer's file.
+
+When set to `auto`, deoplete-go will try to set `GOOS` by checking the file
+name for `name_<OS>.go`.  If not found, the first 10 lines will be checked for
+first `// +build <OS>` directive.  If the file's OS doesn't match your OS (e.g.
+`file_darwin.go` while on `linux`), `CGO_ENABLED=0` will also be set.
+
+**Note:** You may need to run `gocode close` for this to take effect.
```

Comment:
Maybe.  I haven't re-checked what caused the issue for me.  But, the note can just be removed.